src/electron/electron_main.webpack.js

```diff
@@ -13,30 +13,37 @@
 // limitations under the License.
 
 const path = require('path');
+const webpack = require('webpack');
 
-module.exports = {
-  entry: './src/electron/index.ts',
-  target: 'electron-main',
-  node: {
-    __dirname: false,
-    __filename: false
-  },
-  mode: 'production',
-  devtool: 'inline-source-map',
-  module: {
-    rules: [
-      {
-        test: /\.tsx?$/,
-        use: 'ts-loader',
-        exclude: /node_modules/,
-      },
+module.exports = (env) => {
+  return {
+    entry: './src/electron/index.ts',
+    target: 'electron-main',
+    node: {
+      __dirname: false,
+      __filename: false
+    },
+    mode: 'production',
+    devtool: 'inline-source-map',
+    module: {
+      rules: [
+        {
+          test: /\.tsx?$/,
+          use: 'ts-loader',
+          exclude: /node_modules/,
+        },
+      ],
+    },
+    resolve: {
+      extensions: [ '.tsx', '.ts', '.js' ],
+    },
+    plugins: [
+      new webpack.DefinePlugin({
+        GO_NETWORK_STACK: JSON.stringify(!!env.GO_NETWORK_STACK),
```

Comment:
I think it's easier to understand if we keep the variable constant across the code (rather than as string and as boolean):
```suggestion
        NETWORK_STACK: JSON.stringify(env.NETWORK_STACK),
```

Perhaps we name it NETWORK_STACK and require `=="go"`. That may be a bit easier to read.

Reply:
Done in 1d26153.